openstack/clustering/v1/webhooks/requests.go

```diff
@@ -0,0 +1,43 @@
+package webhooks
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// TriggerOpts Webhooks request parameters
+type TriggerOpts struct {
+	// V (Optional)	query	string	The webhook implementation version requested.
+	V string `q:"V, omitempty"`
+
+	// params (Optional)	query	object	The query string that forms the inputs to use for the targeted action.
```

Comment:
The above comment looks like it was copied and pasted straight from the API docs (with odd spacing included). Inline documentation is turned into official reference documentation, so we want to make sure it's readable. For the above comment, how about something like:

```
Params defines the input to use for the targeted action.
```

Reply:
Removed the comment. Found out that having comment and blank lines prevents go tool to properly format